File: single_player/tutorial/c/onboarding.c
#include <stdlib.h>
#include <stdio.h>
#include <string.h>

int main()
{
    while (1) {
        char enemy1[256]; // name of enemy 1
        scanf("%s", enemy1);
        int dist1; // distance to enemy 1
        scanf("%d", &dist1);
        char enemy2[256]; // name of enemy 2
        scanf("%s", enemy2);
        int dist2; // distance to enemy 2
        scanf("%d", &dist2);

        printf("%s\n", (dist1 < dist2) ? enemy1 : enemy2);
    }
}

File: single_player/tutorial/js/onboarding.js
for(;;) {
    var enemy1 = readline(),    // First enemy name
        dist1 = +readline(),    // First enemy distance from player
        enemy2 = readline(),    // Second enemy name
        dist2 = +readline();    // Second enemy distance from player

    // Figure out which of the two enemies is closest to player and target them
    print((dist1 < dist2) ? enemy1 : enemy2);
}